docs/beyond_basics/101-182-catalog.rst

```diff
@@ -3,7 +3,573 @@
+(such as ``name``, ``doi``, ``url``, ``description``, ``license``, ``authors``, and more), as well as fields that support identification, versioning,
+dataset context and linkage, and file tree specification.
+
+The process of generating a catalog, after metadata entry validation, involves:
+
+1. aggregation of the provided metadata into the catalog filetree, and
+2. generating the assets required to render the user interface in a browser.
+
+.. figure:: ../artwork/src/catalog/datalad_catalog_howitworks.svg
+
+The output is a set of structured metadata files, as well as a `Vue.js <https://vuejs.org/>`_-based browser interface that understands how to render
+this metadata in the browser. What is left for the user is to host this content on their platform of choice and to serve it for the world to see!
+
+
+The DataLad-based workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The DataLad ecosystem provides a complete set of free and open source tools that, together, provide full control over dataset/file access
+and distribution, version control, provenance tracking, metadata addition/extraction/aggregation, and catalog generation.
+
+.. figure:: ../artwork/src/catalog/datalad_catalog_pipeline.svg
+
+- DataLad itself can be used for decentralised management of data as lightweight, portable and extensible representations.
+- DataLad MetaLad can extract structured high- and low-level metadata and associate it with these datasets or with individual files.
+- And at the end of the workflow, DataLad Catalog can turn the structured metadata into a user-friendly data browser.
+
+.. importantnote:: DataLad Catalog also operates independently
+
+   Since it provides its own schema in a standard vocabulary,
+   any metadata that conforms to this schema can be submitted
+   to the tool in order to generate a catalog. Metadata items
+   do not necessarily have to be derived from DataLad datasets,
+   and the metadata extraction does not have to be conducted via
+   DataLad MetaLad. Even so, the provided set of tools can be
+   particularly powerful when used together in a distributed
+   (meta)data management pipeline.
+
+Step-by-Step
+^^^^^^^^^^^^
+
+Installing DataLad Catalog
+""""""""""""""""""""""""""
+
+Let's dive into it and create our own catalog! We'll start by creating and activating a new and empty virtual environment:
+
+.. code-block:: bash
+
+   $ python -m venv my_catalog_env
+   $ source my_catalog_env/bin/activate
+
+Then we can install ``datalad-catalog`` with ``pip``. This process will also install ``datalad`` and other dependencies:
+
+.. code-block:: bash
+
+   $ pip install datalad-catalog
+
+After that, you can check the installation by running the ``datalad catalog`` command with the ``--help`` flag:
+
+.. runrecord:: _examples/DL-101-182-101
+   :language: console
+   :workdir: catalog
+   :cast: catalog_basics
+   :notes: Let's test the installation and look at the help information
+   
+   $ datalad catalog --help
```

Comment:
In the printed book, this will be multiple (physical) pages of help output. I suspect that it is not needed to include this in full. You could consider adding a `:lines: field to pick a meaningful, short snippted (e.g. `:lines: 1, 24-50`, numbers are random).